acceptance/lib/helper.rb

```diff
@@ -290,8 +290,7 @@ def https_request(url, request_method, cert, key, body = nil)
   end
 
   def hup_server(host = master, timeout = 30)
-    pidfile = on(host, 'puppet master --configprint rundir').stdout.chomp + '/puppetserver' 
-    pid = on(host, "cat #{pidfile}").stdout.chomp
+    pid = on(host, "pgrep -f #{options['puppetservice']}").stdout.chomp
```

Comment:
@haus Works both ways for me - puppet-server and puppetserver - on a PE Puppet Server package build that has "puppet-server-release.jar".  Maybe that works because the path to the jar has "/puppetserver/" in it even though the immediate name of the jar has "/puppet-server-release.jar"?  But, yeah, maybe including the dash here would be better.